api/v1alpha1/types_etcdcopybackupstask.go

```diff
@@ -93,3 +97,7 @@ type EtcdCopyBackupsTaskStatus struct {
 	// +optional
 	LastError *string `json:"lastError,omitempty"`
 }
+
+func (e *EtcdCopyBackupsTask) GetCopyBackupsJobName() string {
```

Comment:
```suggestion
func (e *EtcdCopyBackupsTask) GetJobName() string {
```
The struct on which you are calling the method already is named `EtcdCopyBackupsTask`, there is no need to repeat that in the method as well.